pkg/controllers/vdb/version_reconciler.go

```diff
@@ -69,6 +71,20 @@ func (v *VersionReconciler) Reconcile(ctx context.Context, _ *ctrl.Request) (ctr
 		return ctrl.Result{Requeue: true}, nil
 	}
 
+	if vmeta.UseVClusterOps(v.Vdb.Annotations) {
```

Comment:
We should add a test case for this part

Reply:
we have e2e leg 6 for this part

Reply:
Yes sorry by test case I meant unit test